packages/nextjs/test/integration/package.json

```
@@ -7,7 +7,7 @@
   },
   "dependencies": {
     "@sentry/nextjs": "file:../../",
-    "next": "latest",
+    "next": "10.1.3",
```

```suggestion
    "next": "10.x",
```

In case they release patch versions.